### PR TITLE
Feat quests page

### PR DESCRIPTION
**Pull request by Annie04082020**
1. Add quest page:
- Add background from Figma
- Add Header
- Add slogan
- Change node color, line style within unlock and complete status
2. Deleted swagger part in docker-compose file

### CODE REVIEW

File: app/src/components/QuestMap.vue [hunk 1]
```diff
@@ -45,30 +75,31 @@ async function getProgress() {
 }
 
 const genOption = (nodes, edges) => ({
-  rendener: 'svg',
+  render: 'svg',
```
Comment (lancatlin) [inline app/src/components/QuestMap.vue:41]: I think it should be `renderer`?

File: app/src/components/QuestMap.vue [hunk 2]
```diff
@@ -130,12 +110,17 @@ function getOption(quests, progress) {
     completed: progress[node._id]?.completed || false,
     unlocked: node.requirements.every((req) => progress[req]?.completed),
   }))
-
-  console.log(nodes)
-
+  const lineAppearence = (node) => {
+    const { unlocked } = node
```
Comment (lancatlin) [inline app/src/components/QuestMap.vue:96]: Consider adding error handling when node is not found (should not happen but for safety)

**Review by lancatlin — APPROVED** [review]
Well done!

File: app/src/components/QuestMap.vue (hunk 1 above)
Comment (Annie04082020) [reply app/src/components/QuestMap.vue:41]: Sorry that I deleted it because ChatGPT recommand me to do it

**Pull request merged**